manifests/istio/istio/overlays/https-gateway/kustomization.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- github.com/kubeflow/manifests/istio/istio/base?ref=master
+
+patchesStrategicMerge:
+- kf-istio-resources.yaml
+
+configMapGenerator:
+- name: istio-parameters
+  behavior: merge
+  envs:
+  - params.env
+configurations:
+- params.yaml
```

Comment:
please add
```yaml
generatorOptions:
  disableNameSuffixHash: true
```
In this way, the configmap will not be suffixed, so you can refer it for `$(gatewaySelector)`

Reply:
do we need to refer it at all?